### PR TITLE
Support getResult on projection

### PR DESCRIPTION
**Pull request by set4812**
Hello,

Currently, the library can get the only state from the projection. I have added support to get result for projection. I would be grateful if you would merge this and publish a package

Thanks
Łukasz

### CODE REVIEW

File: lib/httpClient/projections/getResult.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import debugModule from 'debug';
+import assert from 'assert';
+import axios from 'axios';
+
+const debug = debugModule('geteventstore:getResult');
+const baseErr = 'Get Projection State - ';
```
Comment (mmuller99) [inline lib/httpClient/projections/getResult.js]:
```suggestion
const baseErr = 'Get Projection Result - ';
```

**Comment by set4812** [comment]
Commited

**Comment by mmuller99** [comment]
Thanks.  Apologies but can you please add the TS definition as well?

**Comment by set4812** [comment]
Added.

**Comment by set4812** [comment]
When you publish these changes? Any ETA?

**Comment by mmuller99** [comment]
It will be published soon, most likely tomorrow.

**Comment by mmuller99** [comment]
3.2.0 has been published with your changes

**Pull request merged**